core/src/conn.rs

```diff
@@ -165,6 +211,20 @@ impl MongoConnection {
 
         load_mongosqltranslate_library();
 
+        let is_libmongosqltranslate_compatible_with_driver_version =
+            if get_mongosqltranslate_library().is_some() {
+                let libmongosqltranslate_version = Self::get_libmongosqltranslate_version()?;
+
+                // TODO where do I put the library version for the logs?
+                dbg!(libmongosqltranslate_version);
```

Comment:
I know I'm supposed to add the library version to the client metadata, but I wasn't sure where I was supposed to add it. Is it supposed to be added to the `ClientOptions.app_name` or `ClientOptions.driver_info`? Or somewhere else?

Reply:
This should be added into the appname.

Reply:
Thanks! Done!